_posts/blog/2018-11-02-el-topo-blindado.md

```diff
@@ -0,0 +1,59 @@
+
+The stated aims behind the creation of the new site are twofold: to deepen the study on the armed struggle in Argentina and to more effectively establish an archive of primary sources that will remain free of access to any interested persons(s). The site itself is divided into four areas: 
+
+-	Documents (the digital archive of OPM primary sources)
+-	Articles (a digital archive of secondary material)
+-	Books (a digital archive of the work produced by the collective in book format)
+-	Index (a listing of the OPM's whose documentation is presented in the site)
+
+The OPM's whose documentation is collected in this site, although far too many to name here, are organized into a classificatory system based on their political ideology/methodology. These are:
+
+-	**OPM Frentistas**: organizations whose social-construction project was based on the unity of action of the different organizations. For example, *Fuerzas Argentinas de Liberación* (FAL) and *Guerrilla of the Liberation Army* (GEL). 
+-	**OPM Marxistas**: OPMs that affirm Marxism (or one of its variants) as the point of departure for their tactics, strategies and political thinking. For example, *Baluarte, Comando Santiago Pampillón* (CSP), *Ejército Guerillero del Pueblo* (EGP), *Ejército Revolucionario del Pueblo* (ERP), *Partido Revolucionario de los Trabajadores* (PRT), among many others.
+-	**OPM Peronistas**: various organizations of Peronist origin. For example, *Acción Revolucionaria Peronista, Resistencia Peronista, Movimiento Revolucionario Peronista*, various student led organizaciones (juventudes), etc. 
+
+The types of primary sources the archive maintains, preserves and disseminates in digital form are: audio recordings, bulletins, comunicado (Statements of policy communicated to the members of the organization), internal documents, pamphlets and books, press articles, visual propaganda and flyers. In addition to these primary sources, under the header Books the site also houses digital copies of eight of the ten books that this collective has produced so far. 
+
+I have chosen to also highlight the previous incarnation(s) of this digital archive project precisely because the engagement in political struggle as it pertains to the archive that Weld describes is also a struggle in time. The research group that maintain this site have evolved from seminars, conferences, classes and local meetings, through a blog platform to the digital archive platform being reviewed here. An archive in this sense is in a constant state of becoming rather than existing as a fixed thing. 
+
+For all of the amazing potential of the technology that makes digital archives possible, there are, however, limitations. 
```

Comment:
What potential? Offer a sentence of what you think the project is successful at: what does it accomplish? What important contribution is it making?